### DISCUSSION TITLE
Sites are no longer sorted alphabetically

### DISCUSSION THREAD

**Discussion by simo-g** [amidaware/tacticalrmm#1117] [Q&A]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04, Debian 10
 - Browser: Tested on chrome and Safari
 - RMM Version (as shown in top left of web UI): 0.13.3

**Installation Method:**
  - [X] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 2.0.3
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
Prior to 0.13.X, the Site listing was always sorted alphabetically.  Since the feature was added to sort Clients, the Sites are no longer sorted.

**To Reproduce**
1. Add multiple sites under a single client
2. The sites are displayed in some order (other than alphabetical, and site_id)
**Expected behavior**
The sites should be sorted in alphabetical order (as they were in 0.12.X

**Screenshots**
![TRMMSites](https://user-images.githubusercontent.com/95686685/166643874-7859185b-d0d7-4775-a707-e9075622f9a0.jpg)

![TRMMSettings](https://user-images.githubusercontent.com/95686685/166643882-77429882-bb05-403b-b89f-ece0cc72c5cc.jpg)

**Additional context**

**Comment by dinger1986** [comment]
Sorting Clients has been a feature for the last 6 months or so but a bug was introduced with some changes to the web ui in version 0.13 this was resolved in 0.13.1 or 0.13.2, please clear your browser cache and try again or try another browser.

**Reply by simo-g** [reply]
Thanks for the reply! I have tried it on multiple browsers multiple computers, including private browsing session and browsers that have never been signed in previously. And just to clarify this is not sorting Clients, it is sorting Sites.

**Reply by dinger1986** [reply]
Yes, I see you discussed with @silversword411 and a feature request was suggested. Opened feature request #1118

**Reply by simo-g** [reply]
Why isn't this a bug? It was working in 0.12, not working in 0.13.

**Reply by dinger1986** [reply]
Because previously there was no sorting on sites, it just defaulted to alphabetical, a feature request should also show it in the dropdown as you want to ideally arrange it in the same way as client, ie failing at the top, alphabetical etc.

**Reply by wh1te909** [reply]
it was actually a bug, we did fix clients sorting but forgot to do sites as well. just pushed a fix in https://github.com/amidaware/tacticalrmm/commit/f0a8c5d732c45478a9694bade161b85d085e3963 will be in next release
